runtime/lua/vim/lsp/_snippet.lua

```diff
@@ -336,25 +330,34 @@ S.transform = P.map(
 )
 
 S.tabstop = P.any(
-  P.map(P.seq(S.dollar, S.int), function(values)
-    return setmetatable({
-      type = Node.Type.TABSTOP,
-      tabstop = values[2],
-    }, Node)
-  end),
-  P.map(P.seq(S.dollar, S.open, S.int, S.close), function(values)
-    return setmetatable({
-      type = Node.Type.TABSTOP,
-      tabstop = values[3],
-    }, Node)
-  end),
-  P.map(P.seq(S.dollar, S.open, S.int, S.transform, S.close), function(values)
-    return setmetatable({
-      type = Node.Type.TABSTOP,
-      tabstop = values[3],
-      transform = values[4],
-    }, Node)
-  end)
+  P.map(
+    P.seq(S.dollar, S.int),
+    function(values)
+      return setmetatable({
+        type = Node.Type.TABSTOP,
+        tabstop = values[2],
+      }, Node)
+    end
+  ),
+  P.map(
+    P.seq(S.dollar, S.open, S.int, S.close),
+    function(values)
+      return setmetatable({
+        type = Node.Type.TABSTOP,
+        tabstop = values[3],
+      }, Node)
+    end
+  ),
+  P.map(
+    P.seq(S.dollar, S.open, S.int, S.transform, S.close),
+    function(values)
```

Comment:
this looks like an improvement

Reply:
This is from the first commit (which is the one to actually look at).